docker/dockerise.sh

```diff
@@ -12,7 +12,11 @@ PUSH=${2:-nopush} # Default won't push to Docker hub
 NODE_VERSION='14'
 POSTGRES_VERSION='12'
 
-IMAGE_TAG="build-${BRANCH_NAME}_$(date +"%Y-%m-%d")_pg-${POSTGRES_VERSION}_node-${NODE_VERSION}"
+# Generate a random ID so Images built on same day with same branch
+# have a unique name
+RANDOM_ID=$(openssl rand -hex 3)
+
+IMAGE_TAG="build-${BRANCH_NAME}_$(date +"%Y-%m-%d")_${RANDOM_ID}"
```

Comment:
I thought it was dumb and a waste of valuable screen real estate to have `_pg-12_node-14` on the end of every tag name. But I've added a random 6-char string instead -- this is so if we make multiple builds from the same branch on the same day they will get unique names (something I've done quite a bit while testing this, and it makes things a lot less confusing)